bootstrap.sh

```diff
@@ -1 +1,12 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl apply -f .infrastructure/statefulSet.yml
+kubectl apply -f .infrastructure/configMap.yml
```

Comment:
The file '.infrastructure/configMap.yml' is referenced here, but it seems like the actual file name might be 'st-configMap.yml'. Verify the file name and update the script accordingly.